src/main/resources/global/90-init-alfresco.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+# the value should contain the "-D" flag if it's a property
+function setJavaOption {
+    JAVA_OPTS="$JAVA_OPTS $1"
+}
+# Function to retrieve password values form the store-password.properties file
+function getProperty {
+    grep -v "^#" "$1" | grep "$2" "$1" | sed -E 's/^[^=]*=(.*)$/\1/'
```

Comment:
This is insufficient to read all valid properties files. `.properties` files can be more complex than you typically use, but it is important to implement [that specification](https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html#load-java.io.Reader-) correctly, so avoid problems when there are (for example backslashes, other kinds of comments, or other separators)